### DISCUSSION TITLE
Welcome to Group-33---2021 Discussions!

### DISCUSSION THREAD

**Discussion by bibabooo123456** [bibabooo123456/Group-33---2021#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bibabooo123456** [comment]
Hi! Let's have a quick meeting to discuss about Gantt Chart (5 min).

- [x] Talk about milestones
- [x] Talk about number of iterations
- [x] Discussions on delivery

**Comment by Fkkkkx** [comment]
yeah!!!!

**Comment by bibabooo123456** [comment]
Hi! Let's have a quick meeting to discuss about Gantt Chart (5 min).

- [ ] Talk about milestones
- [ ] Talk about number of iterations
- [ ] Discussions on delivery

**Comment by bibabooo123456** [comment]
@bibabooo123456 Okkkkkkkkk!
